src/main/java/nl/tudelft/contextproject/gui/PresetController.java

```diff
@@ -69,14 +69,23 @@ private void initialize() {
         applySettings();
         setFactories();
         setActions();
+        
+        tableView.getSortOrder().clear();
+        tableView.getSortOrder().add(presetColumn);
```

Comment:
Looks good overall, but maybe it would be nice to throw these two lines into a method called resort() for example. This further reduces a duplicate code case by one and improves readability.